pkg/subctl/cmd/validate_k8sconfig.go

```diff
@@ -0,0 +1,70 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func validateK8sConfig(cmd *cobra.Command, args []string) {
+	fmt.Println("\nValidating CNI Configuration")
+	fmt.Println("\n----------------------------")
+	configs, err := getMultipleRestConfigs(kubeConfig, kubeContext)
+	exitOnError("Error getting REST config for cluster", err)
+
+	for _, item := range configs {
+		fmt.Println()
+		submariner := getSubmarinerResource(item.config)
+
+		if submariner == nil {
+			fmt.Println(submMissingMessage)
+			continue
+		}
+
+		isSupportedPlugin := false
+		for _, np := range supportedNetworkPlugins {
+			if submariner.Status.NetworkPlugin == np {
+				// SGM: Check the networkPlugin in submariner route-agent matches with route-agent.
```

Comment:
```suggestion
```
We should not have to check that. That's probably a test on the operator (one of those we test via bash). But not here. That would be a bug, and not a problem with configuration,

Reply:
Since its related to the configuration by Submariner-operator added this check as it can save time in manual verification when something goes wrong. I'm fine to remove this if you have a strong opinion on this. 

At the same time, I'd request you to please take a look at the [Epic](https://github.com/submariner-io/submariner-operator/issues/1126) once and let me know if you think something need not be handled before myself/ @maayanf24 starts working on it. Thanks.

Reply:
If the route-agent pods have a different network plugin than the submariner-operator detects, then that's a bug. This could be a regression which should happen in: in https://github.com/submariner-io/submariner-operator/blob/devel/scripts/kind-e2e/lib_operator_verify_subm.sh#L343

I have reviewed all the other items on the Epic and they seem good @sridhargaddam

Reply:
Thanks for checking @mangelajo. I'll remove this check since its already verified in submariner-operator